frontend/src/state/modules/journey/reducer.js

```diff
@@ -9,6 +9,8 @@ const initialState = {
   error: null,
 };
 
+const STATIC_JOURNEYS = process.env.NEXT_PUBLIC_STATIC_JOURNEYS === 'true';
```

Comment:
I miss a comment somewhere telling about this should be removed once journeys are complete in the API. But maybe is too much, up to you.